tietokanta/testidata/tarkastusajot.sql

```diff
@@ -1268,152 +1269,4 @@ INSERT INTO tarkastusreitti (id, pistetyyppi, tarkastusajo, aikaleima, vastaanot
 INSERT INTO tarkastusreitti (id, pistetyyppi, tarkastusajo, aikaleima, vastaanotettu, sijainti, kitkamittaus, lampotila, talvihoito_tasaisuus, lumisuus, kuvaus, kuva, havainnot, polyavyys, sivukaltevuus, kiinteys, laadunalitus, soratie_tasaisuus, sijainti_tarkkuus, tr_numero, tr_alkuosa, tr_alkuetaisyys, tr_loppuosa, tr_loppuetaisyys, liittyy_merkintaan) VALUES (1214, 0, 213, '2017-02-22 07:38:02', '2017-02-22 07:38:05', '01010000005224D7187C501A418431733ADD745B41', null, null, null, null, null, null, '{null}', null, null, null, false, null, 5, null, null, null, null, null, null);
 INSERT INTO tarkastusreitti (id, pistetyyppi, tarkastusajo, aikaleima, vastaanotettu, sijainti, kitkamittaus, lampotila, talvihoito_tasaisuus, lumisuus, kuvaus, kuva, havainnot, polyavyys, sivukaltevuus, kiinteys, laadunalitus, soratie_tasaisuus, sijainti_tarkkuus, tr_numero, tr_alkuosa, tr_alkuetaisyys, tr_loppuosa, tr_loppuetaisyys, liittyy_merkintaan) VALUES (1215, 0, 213, '2017-02-22 07:38:04', '2017-02-22 07:38:05', '01010000005037C36153501A415B169E5AE2745B41', null, null, null, null, null, null, '{null}', null, null, null, false, null, 5, null, null, null, null, null, null);
 INSERT INTO tarkastusreitti (id, pistetyyppi, tarkastusajo, aikaleima, vastaanotettu, sijainti, kitkamittaus, lampotila, talvihoito_tasaisuus, lumisuus, kuvaus, kuva, havainnot, polyavyys, sivukaltevuus, kiinteys, laadunalitus, soratie_tasaisuus, sijainti_tarkkuus, tr_numero, tr_alkuosa, tr_alkuetaisyys, tr_loppuosa, tr_loppuetaisyys, liittyy_merkintaan) VALUES (1216, 0, 213, '2017-02-22 07:38:06', '2017-02-22 07:38:07', '010100000073A7773C43501A417D5CBC62E4745B41', null, null, null, null, null, null, '{null}', null, null, null, false, null, 5, null, null, null, null, null, null);
-INSERT INTO tarkastusreitti (id, pistetyyppi, tarkastusajo, aikaleima, vastaanotettu, sijainti, kitkamittaus, lampotila, talvihoito_tasaisuus, lumisuus, kuvaus, kuva, havainnot, polyavyys, sivukaltevuus, kiinteys, laadunalitus, soratie_tasaisuus, sijainti_tarkkuus, tr_numero, tr_alkuosa, tr_alkuetaisyys, tr_loppuosa, tr_loppuetaisyys, liittyy_merkintaan) VALUES (1217, 0, 213, '2017-02-22 07:38:08', '2017-02-22 07:38:10', '01010000007F6A059B28501A413347A0CCE8745B41', null, null, null, null, null, null, '{null}', null, null, null, false, null, 5, null, null, null, null, null, null);
-INSERT INTO tarkastusreitti (id, pistetyyppi, tarkastusajo, aikaleima, vastaanotettu, sijainti, kitkamittaus, lampotila, talvihoito_tasaisuus, lumisuus, kuvaus, kuva, havainnot, polyavyys, sivukaltevuus, kiinteys, laadunalitus, soratie_tasaisuus, sijainti_tarkkuus, tr_numero, tr_alkuosa, tr_alkuetaisyys, tr_loppuosa, tr_loppuetaisyys, liittyy_merkintaan) VALUES (1218, 0, 213, '2017-02-22 07:38:10', '2017-02-22 07:38:13', '01010000005FCD6A5208501A41E69609A4EE745B41', null, null, null, null, null, null, '{null}', null, null, null, false, null, 5, null, null, null, null, null, null);
```

Comment:
Näitä pisteitä ei tarvita, koska ajoon liittyvä geometrisointiongelma tapahtui nimenomaan ajon alussa, eikä ajolle ole muita testejä.